tmt/steps/execute/__init__.py

```diff
@@ -751,7 +751,7 @@ def _process_results_partials(
         :returns: list of results.
         """
 
-        test, guest = invocation.test, invocation.guest
+        test, _ = invocation.test, invocation.guest
```

Comment:
Is there a reason to keep `invocation.guest` here?
```suggestion
        test = invocation.test
```

Reply:
Addressed in 0985617b